docs/getting-started/macos-instructions.md

```diff
@@ -0,0 +1,86 @@
+# Getting Started with Spark .NET on MacOS
+
+These instructions will show you how to run a .NET for Apache Spark app using .NET Core on MacOSX.
+
+## Pre-requisites
+
+- Download and install the following: **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** 
+- Download and install Java using 
+    ```shell
+    brew cask install java
```

Comment:
We should be installing java8.  Doesn't the current command run [this](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/java.rb) which installs java12?

Reply:
Also since homebrew isn't shipped with MacOS by default and is a 3rd party package management solution, I would add some note somewhere regarding this and that these `brew` commands assumes it's installed.